src/components/Select.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+
+const Select = ({ disabled, ...props }) => (
+  <div
+    css={css`
+      height: 28px;
```

Comment:
We might want to consider using `rem` for this so that it will scale with font-size settings.

```suggestion
      height: 1.75rem;
```